parlai/core/torch_agent.py

```diff
@@ -75,6 +77,8 @@ def __init__(self, opt, shared=None):
         self.history_tokens = opt['history_tokens']
         self.history_dialog = opt['history_dialog']
         self.history_replies = opt['history_replies']
+        self.model = None
+        self.optimizer = None
```

Comment:
I don't think we should set these to None

Reply:
Should I leave them out and check for both 'model' and 'optimizer' in the save function? Or is it safe to assume that the child class will have a 'model' field?

Reply:
Oh I already was checking for 'model'. I'll just remove these.